source/standards/pair-programming.html.md.erb

```diff
@@ -0,0 +1,81 @@
+A pair working independently, and able to self-review their work, is going to context-shift considerably less than a person working alone.
+[Reducing the amount of context-shifting](https://blog.ninlabs.com/2013/01/programmer-interrupted/) will enable the pair to focus, and therefore be productive, for longer periods of time.
+
+Pair programming is a practical way of building collective code ownership between members of the team.
+Distributing the understanding of the service being built means that teams are more resilient to people being ill, on leave, or leaving GDS, and as such are less likely to have delivery hiccups from sudden loss of expertise.
+
+Pairing people of [different levels of expertise](https://www.youtube.com/watch?v=lvs7VEsQzKY&t=317s) is an effective way of mentoring and fostering career development.
+For example, pairing someone new to GDS with an experienced developer within the team will benefit both; upskilling the new joiner, and reinforcing the knowledge of the experienced person by teaching them.
+
+## Working effectively as a pair
+
+Pair programming is a development skill in the same way that test-driven development is a skill.
+It’s also a social and collaborative way of working and the way a pair works can vary depending on who you are pairing with.
+We provide some of the following advice to help pairs work effectively together.
+
+### Rotate pairs regularly
+Pairs should rotate regularly at a cadence that makes sense for the team.
+This might be after a single piece of work or after a fixed period of time.
+Rotating pairs spreads knowledge, increases team resilience, and builds ownership of the codebase.
+
+### Discuss how to pair together
+Pairs should agree on the way that they will pair before starting the work itself.
+Your pair might favour ping-pong programming (where one person writes a test, the other person makes it pass and then writes the next test) or strong-style pair programming (your ideas need to enter the codebase through the other person’s input).
+This also includes agreeing when to take breaks.
+
+### Take breaks
+Working together and staying focused for extended periods of time can be tiring.
+Pairs should make sure that they take regular breaks and where possible work in small iterative batches rather than large changes.
+You might find the Pomodoro technique useful or one of the many pair/mob programming timers available online.
+Have a debrief at the end of the day if it would be helpful, and raise any recurring issues at your team’s retrospectives or with your tech lead.
+
+### Pair where appropriate
+Pairing is not suitable for every type of work, so use your discretion within the team.
+You might choose not to pair on automated refactorings or work that would be described as Clear within the Cynefin model.
```

Comment:
It might be helpful to add a link to something defining the Cynefin model?

Reply:
Good shout, done.